aopy/data/base.py

```diff
@@ -888,16 +883,11 @@ def align_recoring_drives(neuropixel_drive, drive2, subject):
     else:    
         angle = 0
 
-    # Align neuropixel drive to drive2 based on trig - neuropixel insert coordinates are centered by default, but adjust if necessary
-    np_offset = (np.max(np_drive_ch_pos, axis=0) + np.min(np_drive_ch_pos, axis=0))/2 # Find center of drive 2
-    aligned_np_drive_coordinates_x = (np.cos(angle)*(np_drive_ch_pos[:,0] - np_offset[0])) - (np.sin(angle)*(np_drive_ch_pos[:,1] - np_offset[1]))
-    aligned_np_drive_coordinates_y = (np.sin(angle)*(np_drive_ch_pos[:,0] - np_offset[0])) + (np.cos(angle)*(np_drive_ch_pos[:,1] - np_offset[1]))
-    aligned_np_drive_coordinates = np.concatenate((aligned_np_drive_coordinates_x[:,None], aligned_np_drive_coordinates_y[:,None]), axis=1)
+    # Load ch map for neuropixel drive and drive 2
+    np_drive_ch_pos, recording_sites , _ = load_chmap(drive_type=neuropixel_drive, theta=-np.rad2deg(angle))    
+    drive2_ch_pos, acq_elecs , _ = load_chmap(drive_type=drive2)
 
-    # Compute offset to align with drive 2
-    drive2_offset = (np.max(drive2_ch_pos, axis=0) + np.min(drive2_ch_pos, axis=0))/2 # Find center of drive 2
-    
-    return (aligned_np_drive_coordinates + drive2_offset), recording_sites
+    return np_drive_ch_pos + neuropixel_drive_offset, drive2_ch_pos + drive2_offset, recording_sites, acq_elecs
```

Comment:
i think this np_drive_ch_pos + offset functionality would be better handled by the load_chmap function. it could have its own 'offset' parameter. i also think that function should by default center the channel mappings on the offset (i.e. remove the rotation_offset and then add the new offset)

Reply:
I didn't include this because load_chmap takes in a rotation_offset parameter and I thought it would be confusing.

Reply:
If we should center the channel mappings on the offset we should also center on 0 by default. Either way it is altering the existing coordinates to be meaningless so we might as well remove all ambiguity.

I don't understand what you mean by remove the rotation_offset and add a new offset, I'm not applying a rotation_offset.

Reply:
i agree centering at (0,0) makes sense by default. i think that should happen in load_chmap if possible so others can make utilities like yours without having to redo that logic. 

the rotation_offset is currently only used for rotation but the maps are left at their own coordinates (sometimes not centered on (0,0)). i suggest rotation_offset could also be subtracted permanently in load_chmap from the positions returned by `map_acq2pos`. then everything would be centered at (0,0) by default.